### PR TITLE
feat: add RocketCleaner

### PR DESCRIPTION
**Pull request by Sanjit1**
Hi! This is a pull request to add @miladdavoodi 's app: RocketCleaner
The apps repository is over [here](https://github.com/miladdavoodi/RocketCleaner), and the proof that it is using electron is over [here](https://github.com/miladdavoodi/RocketCleaner/blob/cdde565cff554236c3ad4396be9c44f3ecbd3193/package.json#L77). 
https://img.shields.io/badge/Keep%20Koding%2C%20Best%20regards-Sanjit-accff2

### CODE REVIEW

File: apps/rocketcleaner/rocketcleaner.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: RocketCleaner
+description: 'MAC OS hdd & ram.'
```
Comment (HashimotoYT) [inline apps/rocketcleaner/rocketcleaner.yml]: These description doesn't answer for questions "what's, why" this application does.

Comment (Sanjit1) [reply apps/rocketcleaner/rocketcleaner.yml]: Ummm Ok but this is not my app, So I will ask @miladdavoodi to answer that and will make the changes

Comment (miladdavoodi) [reply apps/rocketcleaner/rocketcleaner.yml]: Do you mean my app doesn't work?
@HashimotoYT 

@Sanjit1 
#After running the program performs the following tasks

# Empty the Trash on all mounted volumes and the main HDD..
- rm -rfv /Volumes/*/.Trashes/* &>/dev/null
- rm -rfv ~/.Trash/* &>/dev/null
# Clear System Log Files...
- rm -rfv /private/var/log/asl/*.asl &>/dev/null
- rm -rfv /Library/Logs/DiagnosticReports/* &>/dev/null
- rm -rfv /Library/Logs/Adobe/* &>/dev/null
- rm -rfv ~/Library/Containers/com.apple.mail/Data/Library/Logs/Mail/* &>/dev/null
- rm -rfv ~/Library/Logs/CoreSimulator/* &>/dev/null
# npm cache clean
- npm cache clean --force
# Clear Adobe Cache Files...
- rm -rfv ~/Library/Application\ Support/Adobe/Common/Media\ Cache\ Files/* &>/dev/null
# Cleanup iOS Applications...
- rm -rfv ~/Music/iTunes/iTunes\ Media/Mobile\ Applications/* &>/dev/null
# Remove iOS Device Backups...
- rm -rfv ~/Library/Application\ Support/MobileSync/Backup/* &>/dev/null
# Cleanup XCode Derived Data and Archives...
- rm -rfv ~/Library/Developer/Xcode/DerivedData/* &>/dev/null
- rm -rfv ~/Library/Developer/Xcode/Archives/* &>/dev/null
# Cleanup pip cache...
- rm -rfv ~/Library/Caches/pip

#Purge inactive memory... 
- purge

Comment (Sanjit1) [reply apps/rocketcleaner/rocketcleaner.yml]: No. It's just that we need a better description. The description does not answer the question: What does this app do?
or
Why would someone need this app?

But I think you answered the question. So I will go with the description to :
`Empty trash from HDD's, Cleans system log files, IOS Applications, pip Cache, etc.`

[https://img.shields.io/badge/Keep%20Koding%2C%20Best%20regards-Sanjit-accff2](https://www.sanjit.wtf)

Comment (miladdavoodi) [reply apps/rocketcleaner/rocketcleaner.yml]: thank you @Sanjit1

**Comment by Sanjit1** [comment]
I mean just by Updating the branch now I have 30 commits to merge. So... It has been a while lol.

[best-regards](https://www.sanjit.wtf)

**Pull request merged**